app/controllers/posts_controller.rb

```diff
@@ -8,7 +8,7 @@ def index
   end
 
   def new
-    @post = Post.new
+    @post = Post.new(post_type: "needed")
```

Comment:
If we want one of the radio buttons pre-checked, yes. Since FormBuilder calls `@post.post_type`, to figure out which should be checked by default.